### DISCUSSION TITLE
Object counter

### DISCUSSION THREAD

**Discussion by Minglarn** [blakeblackshear/frigate#2671] [Ask A Question]
Hi folks…
Came up with an idea…
I want to count number or persons passing by in a zone.
How would you make an automation to create a counter/sensor that counts number of passing peoples in a zone?

The zone I want to count is named: street_area and i’ve set up an counter witch increments every time a MQTT (frigate/street_area/person) is triggerd. But there seems that this is triggerd every time Frigate finds a new person. I’ve seen in the MQTT message that there is an person counter. 
![image](https://user-images.githubusercontent.com/58695742/150117811-09180941-613a-4e08-84db-6e5d97a68545.png)
Woulnd it be possible to add this person number to a sensor in HA?
This counter has to be reset every midnight .. I’m new to this counter and helper stuff… :slight_smile:

How would you do?

**Comment by blakeblackshear** [comment]
Use the events topic to increment a counter in home assistant with an automation for each `end` event type.

**Reply by Minglarn** [reply]
That's what i'm doing right now. But I've seen that the person is trigged several times during the motion... 
The automation is only triggerd when frigate/street_area/person is broadcasted. 
If 5 persons are detected I would like to increment the counter with 5 ... At the moment it's only incrementing by 1 ...

**Reply by blakeblackshear** [reply]
Make sure you are only counting the messages of type "end" to count each object only once.
